### ISSUE TITLE
Serve openingh.openstreetmap.de over HTTPS

### ISSUE THREAD

**Issue by ypid** [opening-hours/opening_hours.js#203]
My mirror https://openingh.ypid.de/evaluation_tool/ is serving over HTTPS since a few months #172. openingh.openstreetmap.de is outside of my administrate control. Lets see what the best solution is here in the long run.

**Comment by rugk** [comment]
Also, BTW, this is required for getting the location of the user, at least in Chrome.

**Comment by ypid** [comment]
Good point :)

**Comment by rugk** [comment]
Yeah, they have [their reasons](https://scotthelme.co.uk/still-think-you-dont-need-https/) to do so… :smiley:
